src/parser/hunter/beastmastery/modules/core/SpellUsable.tsx

```diff
@@ -10,9 +10,7 @@ class SpellUsable extends CoreSpellUsable {
   lastPotentialTriggerForBarbedShotReset: CastEvent | null = null;
 
   on_byPlayer_cast(event: CastEvent) {
-    if (super.on_byPlayer_cast) {
-      super.on_byPlayer_cast(event);
-    }
+    super.on_byPlayer_cast(event);
     const spellId = event.ability.guid;
```

Comment:
I've had issues with doing this before where TS screams at me (and it giving errors in things shown) but if bumping it to 3.8 fixes that then yay. 
I even tried with optional chaining to no avail.

Reply:
with TS 3.8, it actually gave me an error because the condition was always true :eyes: